osf/management/commands/import_EGAP.py

```diff
@@ -36,7 +36,7 @@ class EGAPUploadException(Exception):
 
 
 def ensure_egap_schema():
-    schema = ensure_schema_structure(from_json('egap-registration.json'))
+    schema = ensure_schema_structure(from_json('egap-registration-2.json'))
```

Comment:
it's kinda super confusing that `egap-registration-2` is "version 3"...

also inconsistent -- `osf-preregistration.json` is version 2 (as is every other schema file without a number), while `osf-preregistration-3.json` is version 3 (there is no `osf-preregistration-2.json`)

would you mind renaming the file to `egap-registration-3.json`, to avoid future confusion?